### PR TITLE
Issue #931 - create new cdi config element

### LINKED ISSUE #931: CDI 2.0: Create a CDI configuration element that doesn't say `cdi12`
The current configuration element is `<cdi12>`. That was probably a mistake so for CDI 2.0 (and onwards) we should use just `<cdi>`.

### PR DESCRIPTION
**Pull request by tevans78**
#build 
fixes #931 
this adds the <cdi> config element but i need to port over some tests as well.

### CODE REVIEW

File: dev/com.ibm.ws.cdi.1.2.weld/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+cdi12=Contexts And Dependency Injection (CDI) V1.2
+cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v1.2.
+
+cdi12.enableImplicitBeanArchives.name=Enable scanning implicit bean archives
+cdi12.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries.
```
Comment (chirp1) [inline dev/com.ibm.ws.cdi.1.2.weld/resources/OSGI-INF/l10n/metatype.properties]: I'm OK with this wording.

File: dev/com.ibm.ws.cdi.2.0.weld/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+cdi=Contexts And Dependency Injection (CDI)
+cdi.desc=Defines the behavior of the Contexts and Dependency Injection (CDI).
+
+cdi.enableImplicitBeanArchives.name=Enable scanning implicit bean archives
+cdi.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries.
```
Comment (chirp1) [inline dev/com.ibm.ws.cdi.2.0.weld/resources/OSGI-INF/l10n/metatype.properties]: I'm OK with this wording.

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Pull request merged**